demo/horace_demo_script.m

```diff
@@ -9,10 +9,16 @@
 %====================================
 %% Generate Horace data files
 %====================================
+demo_dir=pwd;
+indir=demo_dir;     % source directory of spe (or nxspe) files
+par_file=[indir,filesep,'4to1_124.par'];     % detector parameter file
+sqw_file=[indir,filesep,'fe_demo.sqw'];        % output sqw file
+data_source =sqw_file;
```

Comment:
you may want to create temporary file sqw_file in the cwd but par file is definitely in the demo directory so either give them separate directories or ensure all ops take place in the demo directory.

Also comment that sqw_file is designed to be temporary and will be deleted at the end